waiter/integration/waiter/authentication_test.clj

```diff
@@ -27,8 +29,9 @@
 (deftest ^:parallel ^:integration-fast test-token-authentication-parameter-error
   (testing-using-waiter-url
     (when (using-composite-authenticator? waiter-url)
-      (let [authentication-providers (map name (keys (get-in (waiter-settings waiter-url)
-                                                             [:authenticator-config :composite :authentication-providers])))
+      (let [authentication-providers (map name (keys (get-in
```

Comment:
thrading macro `->>` would make this easier to read

Reply:
updated